tests/test_auth.py

```diff
@@ -127,20 +129,20 @@ def test_unicode_name(self):
         new_user = add_authomatic_user(authomatic_user, None)
 
         user = User.query.filter_by(email='test@test.org').first()
-        self.assertEqual(user.last_name, u'Bugn\xed')
-        self.assertEqual(new_user, user)
+        assert user.last_name == u'Bugn\xed'
+        assert new_user == user
 
     def test_callback_validation(self):
         """Confirm only valid urls can be set"""
         client = self.add_client()
         self.login()
-        rv = self.client.post('/client/{0}'.format(client.client_id),
+        response = self.client.post('/client/{0}'.format(client.client_id),
                 data=dict(callback_url='badprotocol.com',
                     application_origins=client.application_origins))
-        self.assertEqual(200, rv.status_code)
+        assert 200 == response.status_code
```

Comment:
I see Paul is a fan of [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) :)